### PR TITLE
Frogger-game

### PR DESCRIPTION
**Pull request by DaniaB24**
###frogger-game

[Demo](https://daniab24.github.io/frogger-game/) |
[Code base](https://github.com/DaniaB24/frogger-game)

### CODE REVIEW

File: submissions/DaniaB24/frogger-game/app.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+// Now write your own player class
+// This class requires an update(), render() and
+// a handleInput() method.
+Player.prototype.update = function () {
+  document.getElementById('score').innerHTML = 
+  `
+    <div>
+      <p>Score:<span>${count}</span></p>
+    </div>
+  `
```
Comment (OleksiyRudenko) [inline submissions/DaniaB24/frogger-game/app.js:59]: The class shouldn't be aware of the specificity of the environment it operates in.

The game engine doesn't offer an appropriate method to us. Let's define a global function for this and pretend it is a part of the game engine.

**Review by OleksiyRudenko — CHANGES_REQUESTED** [review]
@DaniaB24 well done!
A couple of improvements and we're done.
P.S. Please also remove comments.

File: submissions/DaniaB24/frogger-game/app.js (hunk 1 above)
Comment (DaniaB24) [reply submissions/DaniaB24/frogger-game/app.js]: thanks for review! I already fix)

File: submissions/DaniaB24/frogger-game/app.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+    default:
+      break;
+  }
+};
+
+let player = new Player(START_X, START_Y),
+  enemy1 = new Enemy(-100, 60, 100, player),
+  enemy2 = new Enemy(-100, 143, 25, player),
+  enemy3 = new Enemy(-100, 228, 77, player),
```
Comment (OleksiyRudenko) [inline submissions/DaniaB24/frogger-game/app.js:95]: Always add some semantics to "magic numbers". I guess, all `-100` and `100` can be effetively replaced with `-CELL_WIDTH` and `CELL_WIDTH`. Remaining numbers remain mysterious. As a peer developer I would love to understand the logic behind them to be able to add more enemies easily wuthout trying to decipher the logic behind calculations.

**Review by OleksiyRudenko — APPROVED** [review]
@DaniaB24 well done.
Check the comment below re "magic numbers" and take this a learning for similar cases you may deal with.

**Pull request merged**